server/api_design/validate_contracts.py

```
@@ -20,79 +21,88 @@ def resolve_ref(ref):
+
+                # ✅ Validate example against schema
+                validate(instance=example, schema=schema)
+                print(f"  ✅ {method} {path} [{status_code}]")
+            except ValidationError as e:
                 print(
-                    f"  ✅ {endpoint['method']} "
-                    f"{endpoint['endpoint']} [{status_code}]"
+                    f"  ❌ {method} {path} [{status_code}] "
+                    f"schema validation failed -> {e.message}"
                 )
             except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)